scss/initial/_icons.scss

```diff
@@ -3,25 +3,55 @@
 
 
 /// # Herman Icons
+/// Herman doesn't use very many icons internally…
 /// @icons assets/svg/
 /// @group style-icons
 
 
-/// A data attribute for setting an icon.
-/// @group style-icons
+/// ## macro `icons()`
+/// Icons should be added to the html using our
+/// `icon()` utility macro in Nunjucks.
+///
+/// - `name` :: {`string`}<br/>
+///   The string ID used to identify the icon desired.
+/// - `alt=none` :: {`string` | `none`}<br/>
+///   Alternate text for the icon will be visually hidden,
+///   but available to screen-readers.
+/// - `size=none` :: {`string` | `none`}<br/>
+///   Currently supported icon sizes include
+///   `'nav-icon'` and `'footer-logo'`.
+/// - `class=none` :: {`string` | `none`}<br/>
+///   Any classes that should be applied to the icon.
+///
 /// @example njk
 ///   {% import 'utility.macros.njk' as utility %}
-///   {{ utility.icon('menu', size='icon-small') }}
+///   {{ utility.icon('menu', size='nav-icon') }}
```

Comment:
This block is not being displayed. Should it be included in `/// @group style-icons`?